testclients/mysql_client_test.cc

```diff
@@ -489,15 +488,15 @@ static void mct_start_logging(const char *test_case_name) {
     return;
   }
 
-  snprintf(mct_log_file_path, FILE_PATH_SIZE, "%s/%s.out.log",
-           (const char *)tmp_dir, (const char *)test_case_name);
+  std::stringstream mct_log_file_path;
+  mct_log_file_path << tmp_dir << "/" << test_case_name << ".out.log";
```

Comment:
Since this is C++20, consider using `std::format`:
`const std::string str = std::format("{}/{}.out.log", tmp_dir, test_case_name);`

Reply:
I had this idea but I'm afraid not all expected compilers support it